chat.go

```diff
@@ -0,0 +1,162 @@
+/*
+	chat.go -- A minimal Go TCP chat example.
+	Run this like
+		> go run chat.go
+	That will run a TCP chat server at localhost:8080.
+	You can connect to that chat server like
+		> telnet localhost 8080
+	And, of course, others can connect using your IP
+	address like
+		> telnet YOUR-IP-HERE 8080
+	assuming your firewall allows it.
+*/
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"os"
+)
+
+const (
+    CONN_HOST      = "0.0.0.0"
```

Comment:
don't use ALL_CAPS in Go names; use CamelCase

Reply:
exported const CONN_HOST should have comment (or a comment on this block) or be unexported